packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Return the url the application is launched from.
+String initializePlatform() {
+  throw UnimplementedError('Unsupported platfrom');
```

Comment:
nit: add more details to message . this also matches the messages for other config specific files like this

"Attempting to initialize framework for unrecognized platform"

Reply:
done